packages/mount/server.test.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @jest-environment node
+ */
+
+import renderHookServer from '../../utils/renderHookServer'
+import useMount from '.'
+
+describe('useMount', () => {
+  test('callback is not executed', () => {
```

Comment:
:thinking: Does this mean the hook doesn't get called when using node?

Reply:
As far as I can see useEffect and useLayoutEffect are not executed server-side. [Ref](https://reactjs.org/docs/hooks-reference.html#uselayouteffect)

Reply:
Ah, that's interesting. I didn't know 👍